src/SDKs/SqlManagement/Management.Sql/Generated/IRestorePointsOperations.cs

```diff
@@ -24,7 +24,7 @@ namespace Microsoft.Azure.Management.Sql
     public partial interface IRestorePointsOperations
     {
         /// <summary>
-        /// Gets a list of database restore points.
+        /// Gets a collection of restore points that belongs to the database.
```

Comment:
Sounds like the grammar is not quire right here. Can you change it in Swagger back to the original (`Gets a list of database restore points.`)

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/2477